pkgs/applications/misc/dbeaver/default.nix

```diff
@@ -151,5 +151,6 @@ stdenv.mkDerivation rec {
     license = licenses.asl20;
     platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ];
     maintainers = with maintainers; [ jojosch ];
+    mainProgram = "dbeaver";
```

Comment:
```suggestion
```

just change the pname to dbeaver to also match the directory name.

Reply:
Yeah, that's probably the better solution (see also https://repology.org/project/dbeaver/versions vs https://repology.org/project/dbeaver-ce/versions).